src/components/Layouts/styled.tsx

```diff
@@ -44,3 +44,11 @@ export const ComponentWrapper = styled.div`
 export const AuthWrapper = styled(ComponentWrapper)`
   min-height: 100%;
 `;
+
+export const StyledNoMatchLayout = styled.div`
```

Comment:
To zdecydowanie nie tutaj. Layouty to generyczna rzecz a nie coś do zastosowania tylko w 1 komponencie.
Te style już gdzieś mamy i możesz z nich skorzystać jako `StyledElement`